test/jdk/java/awt/print/PageFormat/SetOrient.java

```diff
@@ -25,7 +25,6 @@
  * @bug 4186119: setting orientation does not affect printer
  * @summary Confirm that the clip and transform of the Graphics2D is
  *          affected by the landscape orientation of the PageFormat.
- * @run applet/manual=yesno SetOrient.html
  */
```

Comment:
I see `SetOrient.java` has its sibling `SetOrient.html` where you added `@key printer`.

Since both files contain all the tags except for `@test` in the .java file, you should not remove the `@run` tag from the .java file and add `@key printer` to it.